hnn_core/tests/test_gui.py

```diff
@@ -1174,3 +1176,39 @@ def test_default_smoothing(setup_gui):
         assert gui.viz_manager.figs[figid].axes[0].has_data()
 
     plt.close('all')
+
+
+def test_default_frequencies(setup_gui):
+    """Tests that default min/max frequency are inherited correctly"""
+    gui = setup_gui
+
+    # check that the defaults are the same everywhere after running
+    # the default simulation
+    gui.run_button.click()
+
+    gui_min = gui.fig_default_params['default_min_frequency']
+    viz_min = gui.viz_manager.fig_default_params['default_min_frequency']
+    gui_max = gui.fig_default_params['default_max_frequency']
+    viz_max = gui.viz_manager.fig_default_params['default_max_frequency']
+
+    assert gui_min == viz_min == 10
+    assert gui_max == viz_max == 100
```

Comment:
Everything looks great, the only thing I would change is removing the current default frequency values here. I think it's enough to check that they're consistent. We may want to change the default frequency values in the future, but with this check, we would also have to remember to manually change these lines in the tests.